tests/manager/manager-ctor-wireversion.phpt

```diff
@@ -0,0 +1,22 @@
+--TEST--
+MongoDB\Driver\Manager::__construct(): wire version support
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
```

Comment:
As-is, this only skips when the extension isn't loaded. I think it makes sense to remove `SKIPIF` entirely here, as we do for BSON tests and the like, which obviously require the extension.

The other code in `basic-skipif` (e.g. setting up exception handlers) isn't relevant because we don't actually call any utility functions.